### PR TITLE
Add documentation for Carson integration.

### PR DESCRIPTION
**Pull request by rado0x54**
## Proposed change

 Documentation for the new Carson Integration in Home Assistant.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/home-assistant/pull/31336
- This PR fixes or closes issue:

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/carson.markdown [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+title: Carson
+description: Instructions on how to integrate your Carson.live Account within Home Assistant.
+ha_category:
+  - Camera
+  - Lock
+ha_release: 0.105.0
```
Comment (klaasnicolaas) [inline source/_integrations/carson.markdown]:
```suggestion
ha_release: 0.105
```

File: source/_integrations/carson.markdown [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+The `carson` implementation allows you to integrate your [Carson.live](https://carson.live/) account in Home Assistant.
+
+There is currently support for the following device types within Home Assistant:
+
+- [Camera](#camera)
+- [Lock](#lock)
+
+## Configuration
+
+Go to the integrations page in your config and click on new integration -> Carson.
```
Comment (klaasnicolaas) [inline source/_integrations/carson.markdown]:
```suggestion
Go to the integrations page in your config and click on new **integration** -> **Carson**.
```

File: source/_integrations/carson.markdown [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  type: string
+password:
+  description: The password for accessing your Carson account.
+  required: true
+  type: string
+{% endconfiguration %}
+
+## Camera
+
+Once you have enabled the [Carson integration](/integrations/carson), you can start using the camera platform.
```
Comment (klaasnicolaas) [inline source/_integrations/carson.markdown]: Why should we make a link to the same page where the user already is? 🤔

File: source/_integrations/carson.markdown [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+Cameras support image (snapshot) and video stream functionality.
+
+## Lock
+
+Once you have enabled the [Carson integration](/integrations/carson), you can start using the lock platform. You will see all the doors available to you.
```
Comment (klaasnicolaas) [inline source/_integrations/carson.markdown]: Same

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/carson.markdown (hunk 3 above)
Comment (rado0x54) [reply source/_integrations/carson.markdown]: No reason, I used https://github.com/home-assistant/home-assistant.io/blob/next/source/_integrations/ring.markdown as a template and they do it like this. Fixed now.

File: source/_integrations/carson.markdown (hunk 4 above)
Comment (rado0x54) [reply source/_integrations/carson.markdown]: Fixed.

**Comment by frenck** [comment]
Closing PR, as the parent PR has been closed.

**Pull request closed without merging**